src/libraries/System.Runtime.Extensions/tests/System/Diagnostics/Stopwatch.cs

```diff
@@ -95,6 +96,27 @@ public static void StartNewAndRestart()
             }
         }
 
+        [Fact]
+        public static void DebuggerAttributesValid()
+        {
+            DebuggerAttributes.ValidateDebuggerDisplayReferences(new Stopwatch());
```

Comment:
ValidateDebuggerDisplayReferences returns a string of the evaluated data.  Why not use that rather than using custom private reflection?

Reply:
I didn't notice -- I'll fix that.

Reply:
Cool; I like that answer better than "because I tried and it was broken" :smile: